### PR TITLE
Allow vendors to specify client requirements

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

File: docs/source/metainfo.rst [hunk 1]
```diff
@@ -401,6 +401,63 @@ Depth is specified as ``1`` to match the parent device and
       <firmware depth="1">12345678-1234-1234-1234-123456789012</firmware>
     </requires>
 
+Client Features
+---------------
+
+Newer versions of fwupd can restrict updates depending on the features the
```
Comment (superm1) [inline docs/source/metainfo.rst:407]: I think it's worth mentioning the exact version here.  The tense of "newer" will not make sense at some point.

File: docs/source/metainfo.rst [hunk 2]
```diff
@@ -618,6 +675,9 @@ privacy of the remote client.
 The screenshot will only be shown by the front end client when the device has
 the ``_NEEDS_BOOTLOADER`` flag.
 
+Please also add a ``<client>`` requirement if the update cannot be performed
+without showing the image.
```
Comment (superm1) [inline docs/source/metainfo.rst:679]: or message

File: docs/source/metainfo.rst (hunk 1 above)
Comment (hughsie) [reply docs/source/metainfo.rst:407]: Agree, fixed.

File: docs/source/metainfo.rst (hunk 2 above)
Comment (hughsie) [reply docs/source/metainfo.rst:679]: Fixed.

**Pull request merged**